torch/nn/functional.py

```diff
@@ -1256,7 +1256,7 @@ def _instance_norm(input, running_mean=None, running_var=None, weight=None,
                           eps=eps)
 
 
-def layer_norm(input, normalized_shape, running_mean, running_var,
+def layer_norm(input, normalized_shape, running_mean=None, running_var=None,
```

Comment:
Running stats should really be optional here as they are not typically used in LN